core/datetime/src/commonMain/kotlin/com/adammcneilly/pocketleague/core/datetime/DateUtils.kt

```diff
@@ -32,9 +32,9 @@ object DateUtils {
         utcString: String,
         clock: Clock = defaultClock(),
     ): String {
-        require(isBeforeNow(utcString, clock)) {
-            "getRelativeTimestamp only valid for past dates."
-        }
+//        require(isBeforeNow(utcString, clock)) {
+//            "getRelativeTimestamp only valid for past dates."
+//        }
```

Comment:
I'm not sure why this failed, we may want to look into the call site and figure out if the caller should have used something different?

Reply:
MatchDetailDisplayModel uses this, but sometimes we load matches that haven't happened yet, so we need to reconsider and maybe just return an empty string or null if this is not past?